event/callback.go

```diff
@@ -64,3 +64,14 @@ func unwrapLogData(err error) log.Data{
 
 	return logData
 }
+
+// logData returns whether or not the instance was successfully completed
```

Comment:
is this comment about logData or instanceCompleted?
Could we also rename this to `isInstanceCompleted` .. this seems to be a better name from readability perspective.

Reply:
Yeah whoops this an error from the ol' copy-n-paste